docs/output.md

```diff
@@ -43,339 +49,383 @@ The pipeline is built using [Nextflow](https://www.nextflow.io/) and processes d
   - [QC](#qc)
     - [FastQC](#fastqc)
     - [bamQC](#bamqc)
-    - [MarkDuplicates reports](#markduplicates-reports)
+    - [GATK MarkDuplicates reports](#gatk-markduplicates-reports)
     - [samtools stats](#samtools-stats)
     - [bcftools stats](#bcftools-stats)
     - [VCFtools](#vcftools)
     - [snpEff reports](#snpeff-reports)
     - [VEP reports](#vep-reports)
   - [Reporting](#reporting)
     - [MultiQC](#multiqc)
+- [Pipeline information](#pipeline-information)
 
 ## Preprocessing
 
-Sarek preprocesses raw FastQ files or unmapped BAM files, based on [GATK best practices](https://software.broadinstitute.org/gatk/best-practices/).
-
-BAM files with Recalibration tables can also be used as an input to start with the recalibration of said BAM files, for more information see [TSV files output information](#tsv-files)
+`Sarek` pre-processes raw `FASTQ` files or `unmapped BAM` files, based on [GATK best practices](https://gatk.broadinstitute.org/hc/en-us/sections/360007226651-Best-Practices-Workflows).
 
 ### Map to Reference
 
-#### BWA mem
+#### bwa
 
-[BWA mem](http://bio-bwa.sourceforge.net/) is a software package for mapping low-divergent sequences against a large reference genome.
+[bwa](https://github.com/lh3/bwa) is a software package for mapping low-divergent sequences against a large reference genome.
 
 Such files are intermediate and not kept in the final files delivered to users.
 
-### Mark Duplicates
+#### BWA-mem2
+
+[BWA-mem2](https://github.com/bwa-mem2/bwa-mem2) is a software package for mapping low-divergent sequences against a large reference genome.
 
-#### GATK MarkDuplicatesSpark
+Such files are intermediate and not kept in the final files delivered to users.
```

Comment:
It is a bit unclear for me here, which files are not delivered to the user. The mapped bams? Similar in the BWA section

Reply:
Yes, especially since they're not merged.
But yeah, not very clear